var/spack/repos/builtin/packages/recon/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from os import symlink
+
+
+class Recon(MakefilePackage):
+    """RECON: a package for automated de novo identification of repeat families
+       from genomic sequences
+
+       NOTE: The 1.08 is a patched version from the RepeatModeler developers.
+             The original versions can be found on the homepage."""
```

Comment:
How large is the patch? Is there a separate patch file available? One thing that Spack supports is the ability for packages to require patched versions of their dependencies, so if `repeat-modeler` requires a patched version of `recon`, you can have the `recon` package download a fresh tarball and the `repeat-modeler` package tell it to patch `recon` before installation.

Reply:
I looked for a patchfile but it doesn't seem like they provide it. I'll look at the original source and see if it can build with the same recipe.

Reply:
Updated to include a diff of the RepeatMasker patch and use a variant.